extensions/wikia/UserLogin/UserLoginForm.class.php

```diff
@@ -276,4 +307,18 @@ public function EmptySpamFields() {
 		return false;
 	}
 
+	/**
+	 * Check if user is allowed to register account without email confirmation
+	 *
+	 * @return bool
+	 */
+	public function isAllowedRegisterUnconfirmed() {
```

Comment:
Should this function be removed after this ticket is complete and does it need a `Remove when SOC-217 ABTest is finished` note?